src/main/java/spark/Request.java

```diff
@@ -240,7 +240,7 @@ public String body() {
     private void readBody() {
 		try {
 			bodyAsBytes = IOUtils.toByteArray(servletRequest.getInputStream());
-			body = IOUtils.toString(new ByteArrayInputStream(bodyAsBytes));
+			body = new String(bodyAsBytes);
```

Comment:
I agree, I didn't want to change functionality just yet, just fixing random stuff